src/main/java/com/palantir/gradle/gitversion/Git.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.gradle.gitversion;
+
+import java.util.Map;
+
+interface Git {
```

Comment:
Since there's only one implementation of this interface, and we don't actually pass it around, I'd get rid of this interface entirely and just have the implementation.

Reply:
Thanks. I resolved by removing the interface and just having the implementation.